### DISCUSSION TITLE
Preview bot should use labels or other manual trigger by default

### DISCUSSION THREAD

**Discussion by jni** [chanzuckerberg/napari-hub#449] [Ideas]
The preview bot is a great idea, but it is very distracting to have it comment on each commit push to every PR, most of which are not touching the hub data at all (see e.g. [here](https://github.com/jni/affinder/pull/58#issuecomment-1066047461)). I think the default mode should use a manual trigger, either with labels or with a command like `@napari-hub-bot please preview`.

Thoughts?

**Comment by neuromusic** [comment]
thanks @jni ... @czaki has noted this too:

- https://github.com/chanzuckerberg/napari-hub/discussions/389
- https://github.com/chanzuckerberg/napari-hub/discussions/448

now that we've wrapped up our 0.1.0 effort on `napari-sphinx-theme`, we're going to tackle this, initially by updating the comment in-place (see #422)
